connect/connection.go

```diff
@@ -58,14 +59,14 @@ func successCode(code int) bool {
 	return code >= 200 && code < 300
 }
 
-func callHTTP(verb, path string, body io.Reader, query map[string]string, auth authType) ([]byte, error) {
+func callHTTP(verb, path string, body []byte, query map[string]string, auth authType) ([]byte, error) {
```

Comment:
Please take this change out to a separate PR to make parallel work easier and limit future rebasing.

Reply:
will do

Reply:
done